spec/datetimeformat.html

```diff
@@ -303,6 +305,20 @@ <h1>PartitionDateTimePattern ( _dateTimeFormat_, _x_ )</h1>
             1. Else,
               1. Let _fv_ be an implementation and locale dependent String value representing `"ante meridiem"`.
             1. Add new part record { [[Type]]: `"dayPeriod"`, [[Value]]: _fv_ } as a new element of the list _result_.
+          1. Else if _p_ is equal to `"millisecond"`, then
+            1. Let _msd_ be _dateTimeFormat_.[[FractionalSecondDigits]].
+            1. If _msd_ is greater than 0, then
+              1. Let _v_ be _tm_.[[millisecond]].
+              1. If _msd_ = 1, then
+                1. Let _v_ be floor(_v_ / 100).
+              1. Else if _msd_ = 2, then
+                1. Let _v_ be floor(_v_ / 10).
+              1. Let _nf3Options_ be ObjectCreate(*null*).
+              1. Perform ! CreateDataPropertyOrThrow(_nf3Options_, `"minimumIntegerDigits"`, _msd_).
+              1. Perform ! CreateDataPropertyOrThrow(_nf3Options_, `"useGrouping"`, *false*).
+              1. Let _nf3_ be ? Construct(%NumberFormat%, &laquo; _locale_, _nf3Options_ &raquo;).
```

Comment:
I think I'd prefer if `nf3` is created at the top of the loop, next to the other number-formatters, so all internal number-formatters are created in one spot.

So after step 10, where `nf2` is created, add:
```
11. Let _fractionalSecondDigits_ be _dateTimeFormat_.[[FractionalSecondDigits]].
12. If _fractionalSecondDigits_ is greater than 0, then
  a. Let _nf3Options_ be ObjectCreate(*null*).
  b. Perform ! CreateDataPropertyOrThrow(_nf3Options_, `"minimumIntegerDigits"`, _fractionalSecondDigits_).
  c. Perform ! CreateDataPropertyOrThrow(_nf3Options_, `"useGrouping"`, *false*).
  d. Let _nf3_ be ? Construct(%NumberFormat%, &laquo; _locale_, _nf3Options_ &raquo;).
```